app/views/layouts/application.html.erb

```diff
@@ -22,8 +21,8 @@
       <div>
         <ul class="nav navbar-nav navbar-right">
           <li><%= link_to 'Home', root_path %></li>
-          <li><a href="#">All Students</a></li>
-          <li><a href="#">Blacklist</a></li>
+          <li><%= link_to 'All students', students_path %></li>
+          <li><a href="#contact">Blacklist</a></li>
```

Comment:
Hey - wasn't there a blacklisting path helper here courtesy of Gerard? Similar to:

```
 <li><%= link_to 'All students', students_path %></li>
```